wordpress-mobile/WordPress-Android
# Insights Management screen

This is an intermediate PR that lays down the infrastructure for the new Insights Management screen.

To test:

1. Go to Stats
2. Tap the Insights tab
3. Notice there is a new Manage Insights toolbar button
4. Tap on the button
5. Notice a new (empty) Insights management screen is opened with the X button
6. Tap the X button
7. Notice the screen is closed
8. Switch to a DMWY tab
9. Notice the toolbar button is gone
